src/utils/chartJs/CustomTooltip.jsx

```diff
@@ -110,6 +140,9 @@ class CustomTooltip extends React.Component {
           </a>
           )
         </div>
+        <div className={classes.lockMessage}>
+          {isLocked ? 'Click to unlock' : 'Click to lock'}
+        </div>
```

Comment:
As the [treeherder graphs](https://treeherder.mozilla.org/perf.html#/graphs?series=mozilla-central,1776360,1,10&series=mozilla-central,1775979,1,10&series=mozilla-central,1776166,1,10&selected=mozilla-central,1776166,413555,667080523,10), I think it's nice to provide the user with feedback of how to lock the tooltips. I gave it a gray color in the style to make it less intrusive, and if you still find it too much we can discuss about it.